subsys/zigbee/lib/zigbee_fota/src/zigbee_fota.c

```diff
@@ -497,7 +497,8 @@ void zigbee_fota_signal_handler(zb_bufid_t bufid)
 	/* fall-through */
 	case ZB_BDB_SIGNAL_STEERING:
 		if (status == RET_OK) {
-			k_timer_start(&dev_ctx.alarm, K_NO_WAIT, K_HOURS(24));
+			k_timer_start(&dev_ctx.alarm, K_NO_WAIT,
+				K_HOURS(ZIGBEE_FOTA_IMAGE_DISOVERY_INTERVAL_HRS));
 		}
 		break;
```

Comment:
Should we also do this for reboot signal? Simply start timer if not yet started.
For example, for the router device that has joined the network and is restarted.

Reply:
Why not?
```
Start a timer.
...
Attempting to start a timer that is already running is permitted. The timer’s status is reset to zero and the timer begins counting down using the new duration and period values.
```

Reply:
The `ZB_BDB_SIGNAL_DEVICE_REBOOT` is handled here too (there is a fall-through).

Reply:
You're right, no more changes are needed

Reply:
Does it mean that you approve this PR? :slightly_smiling_face: